build.gradle

```diff
@@ -26,7 +26,7 @@ buildscript {
     }
 }
 
-version = "1.10.0.15-SNAPSHOT"
+version = "1.11.0.1-SNAPSHOT"
```

Comment:
This version is tricky and probably has to be 1.11.0.0 for a release, though there will be one commit with this 1-SNAPSHOT version earlier in time than the 1.11.0.0 release... huh...